pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
@amanohar, i made the minimal change for upgrade to align with OrchestratorVersionHint change. I tried to upgrade, now it will upgrade 1.5.x directly to 1.6.6, and it is SUCCESS. I keep the cluster, if you want to take a look. It is exciting for me to see it work !